app/lib/features/cross_signing/pages/request_created.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:acter/common/themes/app_theme.dart';
+import 'package:atlas_icons/atlas_icons.dart';
+import 'package:flutter/material.dart';
+
+class RequestCreatedPage extends StatelessWidget {
```

Comment:
These views seems like most of its inner components are redundant. I would suggest to make one resusable outer wrapper with its inner component as param or strings.

Reply:
Each dialog have different elements.
It is hard to abstract these dialogs into single wrapper.